.github/workflows/Windows_release.yml

```diff
@@ -48,4 +48,5 @@ jobs:
         run: ./build.cmd NuGet
         env:
           Nuget_Key: ${{ secrets.NUGET_KEY }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
LGTM

Reply:
This is provided automatically by GitHub Actions, right? Or does the user have to configure something specific for this?

Reply:
It is provided automatically